packages/core/core/src/requests/ConfigRequest.js

```diff
@@ -74,37 +74,43 @@ export type ConfigRequest = {
   ...
 };
 
-export async function loadPluginConfig<T: PluginWithLoadConfig>(
+export function loadPluginConfig<T: PluginWithLoadConfig>(
   loadedPlugin: LoadedPlugin<T>,
   config: Config,
   options: ParcelOptions,
 ): Promise<void> {
-  let loadConfig = loadedPlugin.plugin.loadConfig;
-  if (!loadConfig) {
-    return;
-  }
+  return measureAsyncFunction(
```

Comment:
what's the goal of measuring internal functions like this? This would be captured by the normal v8 profiler. I thought the goal of tracing is to capture _high level_ timings for user-facing things like plugins. If we start tracing internal things in Parcel core this gets muddier and harder for end users to understand.

Reply:
The goal is to capture high-level timings, so lower resolution than what the profiler will spit out (& lower or close to none overhead).

We could trace more internal bits though. It should not be granular.

I don't really care about the measurements added in this diff. More about the fixes.

For the `measureAsyncFunction` / `measureFunction` helpers ; they prevent mistakes where `.end` isn't currently called.

That was the case on `packages/core/core/src/Transformation.js`, where you can see I added `measurement.end()` calls that were previously not there. For those branches we had missing traces. Ideally we'd replace all `measurement.end()` calls with function wrappers, since then we can't not call `.end()`.